server/graphql/resolvers.ts

```diff
@@ -29,6 +29,12 @@ const resolvers = {
     clients: (_, __, { dataSources, authenticateUser }): Array<ClientInterface> => authenticateUser().then(() => dataSources.clients.getAll()),
     request: (_, { id }, { dataSources }): RequestInterface => dataSources.requests.getById(Types.ObjectId(id)),
     requests: (_, __, { dataSources }): Array<RequestInterface> => dataSources.requests.getAll(),
+    filterRequests: (_, { filter }, { dataSources, authenticateUser }): Array<RequestInterface> => authenticateUser().then(() => {
```

Comment:
`filterRequests` should be named `filterRequestByClientId`. And the param `filter` should be named `clientId`

Otherwise users might assume that this endpoint can filter by any field on a request